pkg/hub/controllers/slicegateway_controller.go

```diff
@@ -86,6 +86,7 @@ func (r *SliceGwReconciler) Reconcile(ctx context.Context, req reconcile.Request
 		SliceGatewayRemoteSubnet:    sliceGw.Spec.RemoteGatewayConfig.GatewaySubnet,
 		SliceGatewayHostType:        sliceGw.Spec.GatewayHostType,
 		SliceGatewayRemoteNodeIP:    sliceGw.Spec.RemoteGatewayConfig.NodeIp,
```

Comment:
We don't have to reconcile `SliceGatewayRemoteNodeIP` since it is removed right?